### PR TITLE
Add new iFlight 2RAW H743 Target

### PR DESCRIPTION
**Pull request by ot0tot**
Draft target based on manufacturer provided schematic.

### CODE REVIEW

File: configs/IFLIGHT_2RAW_H743/config.h [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+#define UART8_RX_PIN            PE0
+#define I2C1_SCL_PIN            PB6
+#define I2C1_SDA_PIN            PB7
+#define I2C2_SCL_PIN            PB10
+#define I2C2_SDA_PIN            PB11
+#define LED1_PIN                PE3
+#define LED2_PIN                PE4
+#define SPI1_SCK_PIN            PA5
+#define SPI2_SCK_PIN            PB13
+#define SPI3_SCK_PIN            PB3
```
Comment (KarateBrot) [inline configs/IFLIGHT_2RAW_H743/config.h:74]: duplicate

File: configs/IFLIGHT_2RAW_H743/config.h [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+#define SPI3_SCK_PIN            PC10
+#define SPI4_SCK_PIN            PE12
+#define SPI1_SDI_PIN            PA6
+#define SPI2_SDI_PIN            PB14
+#define SPI3_SDI_PIN            PB4
```
Comment (KarateBrot) [inline configs/IFLIGHT_2RAW_H743/config.h:79]: duplicate

File: configs/IFLIGHT_2RAW_H743/config.h [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+#define SPI3_SDI_PIN            PC11
+#define SPI4_SDI_PIN            PE13
+#define SPI1_SDO_PIN            PD7
+#define SPI2_SDO_PIN            PB15
+#define SPI3_SDO_PIN            PB5
```
Comment (KarateBrot) [inline configs/IFLIGHT_2RAW_H743/config.h:84]: duplicate

File: configs/IFLIGHT_2RAW_H743/config.h (hunk 1 above)
Comment (ot0tot) [reply configs/IFLIGHT_2RAW_H743/config.h:74]: Thank you for the comments. I am aware of these duplicate defines, but this is what is shown on the most recent schematic. :) I have marked this as a draft until we get more details from iFlight.

Comment (KarateBrot) [reply configs/IFLIGHT_2RAW_H743/config.h:74]: Okay 😁

Comment (KarateBrot) [reply configs/IFLIGHT_2RAW_H743/config.h:74]: `PC10`, `PC11` and `PC12` seem to be the correct pins according to the new schematic

**Comment by KarateBrot** [comment]
I have the FC and tested it. Seems to work just fine.

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**